app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,22 @@
+class MoviesController < ApplicationController
+  before_action :require_user
+  def index
+    conn = Faraday.new('https://api.themoviedb.org')
+
+    if params[:search_title]
+      response = conn.get("/3/search/movie?api_key=#{ENV['MOVIE_API_KEY']}&language=en-US&page=1&include_adult=false&query=#{params[:search_title]}")
+      @movies = JSON.parse(response.body, symbolize_names: true)[:results]
+      if @movies.count == 20
```

Comment:
It will be interesting to see how we end up testing for this if statement's functionality!